server/reload.go

```diff
@@ -771,14 +785,15 @@ func (s *Server) diffOptions(newOpts *Options) ([]option, error) {
 			// Ignore NoLog and NoSigs options since they are not parsed and only used in
 			// testing.
 			continue
+		case "maxtracedmsglen":
```

Comment:
Above it looks like `max_traced_msg_len`

Reply:
This is the reflect field name in lower case of `Options.MaxTracedMsgLen`, so this is the correct spelling.